tests/unit/services/rollbar-test.js

```diff
@@ -95,4 +95,27 @@ test('registerLogger: does not register logger if disabled', function(assert) {
   };
   service.registerLogger();
   Ember.onerror();
+});
+
+test('config: rollbar passes arguments to checkIgnore callback', function (assert) {
```

Comment:
This is not our responsibility to check if it works. But yeah... we should definitely have a test which will check if a proper config has been passed to the rollbar. That's it.

So, we should check if our options from config has been passed to `rollbarService.get('notifier.options')`